src/Poster/Poster.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { layoutOptions } from "./Poster.levers";
+import addBgImage from "../common/util/addBgImage";
+import { Headline } from "./Poster.Headline";
+import { Content } from "./Poster.Content";
+import { Image } from "./Poster.Image";
+
+/**
+ * Poster Component
+ *
+ * HTML card element
+ */
+
+const PosterRoot = ({
+  bgImage: { filename: bgFilename } = {},
+  className,
+  children,
+  layout,
+}) => {
+  let wrapperClasses;
+  let imageWrapper;
+  let contentWrapper;
+  
+  if (layout === "left") {
+    wrapperClasses = "su-flex su-flex-col su-justify-center md:su-flex-row";
+    imageWrapper =
+      "first:su-min-w-[14rem] su-rs-mb-2 su-mx-auto md:su-rs-mr-4 md:su-mb-0 md:su-ml-0";
+    contentWrapper = "last:su-items-start md:su-flex-grow";
+  } else {
+    wrapperClasses = "su-flex su-flex-col su-items-center";
+    contentWrapper = "last:su-items-center su-text-center";
+    imageWrapper = "first:su-rs-mb-2";
+  }
```

Comment:
I think you're totally on the right track. If you worry about spacing above/below the different parts, it's ok to put those in the subcomponents 😄  (<- not sure if that's what you were concerned about). 

FYI (and you probably already know this), the first/last prefixes and the TW classes with [] are not available for this repo sadly 😢 

Let's check with everyone if we want to include background image in this component for now - don't want to scope creep on ya 😂  Please feel free to post to the adapt channel - let's have a discussion on what background color/text colors to include so you don't end up doing more work than needed.

Reply:
Realized that `first:` and `last:` prefixes do not work within out decanter-react repo for flex styles (margin only but even then it's a bit finicky). 😢  I'll rewrite to include the alignments within the stories and remove the lever. Perhaps we can revisit adding the alignment lever if/when we expand the coverage of `first:` and `last:` prefixes?

Reply:
Or we can keep the layout lever since it's could be seen as image left or center! The body content will be left align unless additional classes are passed to the Content subcomponent.